### DISCUSSION TITLE
Exact numbers in the UI

### DISCUSSION THREAD

**Discussion by alex-jitbit** [plausible/analytics#769] [Anything Else]
![image](https://user-images.githubusercontent.com/33555768/109493270-e6bfa580-7a94-11eb-837a-6c4922a7d95b.png)

How do I get the EXACT number of visitors, not rounded to "2.4k"?

I found no other way but to intercept API calls in dev-tools "network" tab but this is ridiculous ;)

Please add a tooltip or something.

**Comment by metmarkosaric** [comment]
thanks for the feedback @alex-jitbit! we will consider to add the tooltip.

we have the stats API now but you can also use this for quick access: https://plausible.io/api/stats/plausible.io/main-graph (just change the second plausible.io in the url to your domain)

**Comment by RobertJoonas** [comment]
Currently the Top Pages details view shows all numbers over 1000 as 1.2k. Would be nice, if there was also a way to see the exact number of visitors directly in that view. Otherwise, I have to filter the dashboard by a certain page and hover over the number of visitors in the top left.

(feedback via email)
